app/src/main/java/org/oppia/android/app/player/state/itemviewmodel/SelectionInteractionViewModel.kt

```diff
@@ -52,10 +56,17 @@ class SelectionInteractionViewModel private constructor(
       ?: minAllowableSelectionCount
   }
   private val selectedItems: MutableList<Int> = mutableListOf()
+  val enabledItemsList by lazy {
```

Comment:
Does this need to be public? It seems to only be used within this model unless I'm missing something.

Reply:
okay